specification/cognitiveservices/data-plane/LUIS/Runtime/v2.0/LUIS-Runtime.json

```diff
@@ -0,0 +1,338 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LUIS Endpoint API",
+    "version": "v2.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "$ref": "../../../Common/ExtendedRegions.json#/parameters/AzureRegion"
```

Comment:
For GA, what is the region list that should be included?
Should the [BasicRegion](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/Common/BasicRegions.json) or [ExtendedRegion](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/Common/ExtendedRegions.json) sets be used instead?

Reply:
If neither of these fit LUIS' profile, feel free to create your own AzureRegion definitions within the spec.

Reply:
Thanks, still awaiting confirmation.

Reply:
@pcostantini when you've got confirmation, ping and we can merge this.

Reply:
Hi @fearthecowboy ,
Just got confirmation, the ExtendedRegions.json is the one to be used, so no further changes are needed.
Thanks!